### ISSUE TITLE
Some test cases fail occasionally

### ISSUE THREAD

**Issue by kentaro** [matsumoto-r/ngx_mruby#131]
I found such test cases like below that occasionally fail. They don’t always fail, but do sometimes.

case1:

```
$ ONLY_BUILD_NGX_MRUBY=true sh test.sh
2015/08/05 14:29:11 [info] 42733#0: *27 ngx_mruby INFO ngx_mrb_run:806:
mrb_run info: rputs_chain is null and return NGX_OK, client: 127.0.0.1,
server: localhost, request: "GET /nginx_var?name=server_protocol HTTP/1.
0", host: "127.0.0.1"
2015/08/05 14:29:11 [alert] 42733#0: *27 zero size buf in writer t:1 r:0
 f:0 00007FDAB6800020 00007FDAB6800098-00007FDAB6800098 0000000000000000
 140577341440000-0, client: 127.0.0.1, server: localhost, request: "GET
/nginx_var?name=server_protocol HTTP/1.0", host: "127.0.0.1"

(snip)

Fail: ngx_mruby - Nginx::Var [location /nginx_var?name=name]
 - Assertion[2] Failed: Expected to be equal
    Expected: "HTTP/1.0"
      Actual: nil
Total: 35
   OK: 34
   KO: 1
Crash: 0
 Time: 0.023947 seconds
```

case2:

```
$ ONLY_BUILD_NGX_MRUBY=true sh test.sh
2015/08/05 15:15:00 [info] 57959#0: *18 ngx_mruby INFO ngx_mrb_run:806:
mrb_run info: rputs_chain is null and return NGX_OK, client: 127.0.0.1,
server: localhost, request: "GET /headers_out_delete HTTP/1.0", host: "1
27.0.0.1"
2015/08/05 15:15:00 [error] 57959#0: *18 ngx_mruby ERROR ngx_mrb_send_he
ader: status code is 200, but response body is empty.Return NGX_HTTP_INT
ERNAL_SERVER_ERROR, client: 127.0.0.1, server: localhost, request: "GET
/headers_out_delete HTTP/1.0", host: "127.0.0.1"

(snip)

Fail: ngx_mruby [location /headers_out_delete]
 - Assertion[50] Failed: 46
    Expected: "foo"
      Actual: nil
Fail: ngx_mruby - Nginx::Var [location /nginx_var?name=name]
 - Assertion[2] Failed: Expected to be equal
    Expected: "HTTP/1.0"
      Actual: nil
Total: 35
   OK: 33
   KO: 2
Crash: 0
 Time: 0.019233 seconds
```

**Comment by matsumoto-r** [comment]
Thank you for your report. I knew this, and this problem occurred only on MacOSX. So, It may be environmental dependency problem. I'll investigate it.

**Comment by matsumoto-r** [comment]
@kentaro fixed via #133 . Please check it.

**Comment by kentaro** [comment]
Thanks!
